apps/cowswap-frontend/vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "redirects": [
+    {
+      "source": "/((?!#|assets|emergency).+)",
```

Comment:
If it works, this is the secret sauce.

This tells vercel to redirect to `/` UNLESS:
- It starts with `/#`, `/assets` OR `emergency` followed by at least 1 other character.

It uses negative lookahead `!?` for that.

Keep in mind this is NOT a pure regex. It uses instead [`path-to-regex`](https://github.com/pillarjs/path-to-regexp/tree/v6.1.0)

Too bad I can't add comments to this JSON doc :/